### PR TITLE
Felinid genetic power: Extendable claws!

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

Felinid only genetic trait that gives the user extendable and retractable claws.

Will add custom sprites in my next commit

## Why It's Good For The Game

Felinids get a bit more ways to stand out without it being some garbage people are going to pointlessly argue over like chocolate, claws are super dope

## Changelog
:cl:
add: New mutation: claws!
/:cl:

### CODE REVIEW

**Comment by Qustinnus** [comment]
did you ask for permission good sir

File: code/datums/mutations/body.dm [hunk 1]
```diff
@@ -487,3 +487,96 @@
 /datum/mutation/human/headless/proc/abortattachment(datum/source, obj/item/bodypart/new_limb, special) //you aren't getting your head back
 	if(istype(new_limb, /obj/item/bodypart/head))
 		return COMPONENT_NO_ATTACH
+
+/datum/mutation/human/claws //not a spell, so doesn't use the "power" variable
+	name = "Claws"
+	desc = "An awakened mutation found in felinids that reshape the bone structure of the wrist to include claws."
+	quality = POSITIVE
+	text_gain_indication = "<span class='warning'>YThere is a sharp pain in your wrists!</span>"
```
Comment (Qustinnus) [inline code/datums/mutations/body.dm]: ythere

**Comment by Ty-the-Smonk** [comment]
no

**Comment by wesoda25** [comment]
>without it being some garbage people are going to pointlessly argue over

bold assumption, what isnt argued about considering them

**Comment by tralezab** [comment]
denied by maintainers, rip wolverine

**Comment by PKPenguin321** [comment]
too bad, this actually isn't terrible as a race-specific gene

**Comment by trollbreeder** [comment]
maybe if it was a mutation for all species

**Comment by tralezab** [comment]
if oranges sees this he has to approve this really cool race specific gene ;)

**Comment by Tharcoonvagh** [comment]
Ew

**Comment by wesoda25** [comment]
I think you should try and get this merged again, it would be good content.

**Pull request closed without merging**